mobile-security-framework/info.json

```diff
@@ -0,0 +1,701 @@
+          "description": "Specify the Hash of the Scan File."
+        }
+      ],
+      "output_schema": {
+        "id": "",
+        "@id": "",
+        "cVEs": [],
+        "file": {
+          "id": "",
+          "@id": "",
+          "size": "",
+          "uuid": "",
+          "@type": "",
+          "assignee": "",
+          "filename": "",
+          "metadata": [],
+          "mimeType": "",
+          "thumbnail": "",
+          "uploadDate": ""
+        },
+        "name": "",
+        "type": "",
+        "uuid": "",
+        "@type": "",
+        "tasks": [],
+        "alerts": [],
+        "assets": [],
+        "owners": [],
+        "people": [],
+        "tenant": {
+          "id": "",
+          "@id": "",
+          "name": "",
+          "role": "",
+          "uuid": "",
+          "@type": "",
+          "region": "",
+          "license": "",
+          "industry": "",
+          "tenantId": "",
+          "createDate": "",
+          "createUser": "",
+          "modifyDate": "",
+          "modifyUser": "",
+          "description": "",
+          "isDedicated": "",
+          "allowRemoteMMDModification": ""
+        },
+        "@context": "",
+        "assignee": "",
+        "comments": [],
+        "conflict": "",
+        "warrooms": [],
+        "incidents": [],
+        "createDate": "",
+        "createUser": {
+          "id": "",
+          "@id": "",
+          "name": "",
+          "uuid": "",
+          "@type": "",
+          "avatar": "",
+          "userId": "",
+          "userType": "",
+          "createDate": "",
+          "createUser": "",
+          "modifyDate": "",
+          "modifyUser": ""
+        },
+        "indicators": [],
+        "modifyDate": "",
+        "modifyUser": {
+          "id": "",
+          "@id": "",
+          "name": "",
+          "uuid": "",
+          "@type": "",
+          "avatar": "",
+          "userId": "",
+          "userType": "",
+          "createDate": "",
+          "createUser": "",
+          "modifyDate": "",
+          "modifyUser": ""
+        },
+        "recordTags": [],
+        "userOwners": [],
+        "workspaces": [],
+        "description": "",
+        "tenantRecordId": "",
+        "vulnerabilities": []
+      }
+    },
+    {
+      "operation": "generate_json_report",
+      "title": "Generate JSON Report",
+      "annotation": "generate_json_report",
+      "description": "Generate JSON Report of the Scan Result",
+      "category": "investigation",
+      "is_config_required": true,
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "hash",
+          "title": "Hash",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the Hash of the Scan File.",
+          "description": "Specify the Hash of the Scan File."
+        }
+      ],
+      "output_schema": {}
```

Comment:
output schema missing ?

Reply:
The outputs of the this actions are different for every file and are very large in size, So after discussion it's decided to leave it as blank